ocs_ci/deployment/acm.py

```diff
@@ -0,0 +1,192 @@
+"""
+All ACM related deployment classes and functions should go here.
+
+"""
+import logging
+import subprocess
+import shlex
+import pexpect
+
+from ocs_ci.framework import config
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.constants import SUBMARINER_GATEWAY_PROMPT
+from ocs_ci.ocs.exceptions import (
+    CommandFailed,
+    DRPrimaryNotFoundException,
+    InteractivePromptException,
+)
+from ocs_ci.utility.utils import run_cmd
+
+
+logger = logging.getLogger(__name__)
+
+
+def run_subctl_cmd(cmd=None):
+    """
+    Run subctl command
+
+    Args:
+        cmd: subctl command to be executed
+
+    Returns:
+
+    """
+    cmd = " ".join("subctl", cmd)
+    out = run_cmd(cmd)
+    logger.info(out)
```

Comment:
return run_cmd(cmd)

logging is done in run_cmd I think

Reply:
done